src/cephadm/tox.ini

```diff
@@ -49,7 +49,7 @@ deps =
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm.py cephadmlib}
-    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 11'
+    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 7'
```

Comment:
@guits there's another one of these in pybind/mgr/cephadm that needs to be updated as well
```
flake8: commands[1] /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr> bash -c 'test $(git ls-files cephadm | grep ".py$" | grep -v tests | xargs grep "docker.io" | wc -l) == 13'
flake8: exit 1 (0.01 seconds) /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr> bash -c 'test $(git ls-files cephadm | grep ".py$" | grep -v tests | xargs grep "docker.io" | wc -l) == 13' pid=922776
flake8: FAIL ✖ in 8.82 seconds
```